src/components/Dropdown/dropdown.module.scss

```diff
@@ -1,44 +1,65 @@
-$dropdown-hover: #d7d6d6;
+$dropdown-hover: #d0e7ff;
 $white: #fff;
 $theme-blue: #1d1283;
 $theme-active: #87d870;
 $black: #000;
 
 .dropdown {
-    width: 180px;
-    background-color: $white;
-    border: 2px solid $black;
-    border-radius: 0.5rem;
+    width: 150px;
```

Comment:
I think we should stick to one unit either rem or Px, and since we are using rem let's stick to rem. px is suitable for those css properties where the value will be same for all screen size.

Reply:
The width stays same. I will attach a video.